src/main/java/com/carina/methods/SauceDemo/commons/CartBaseScreen.java

```diff
@@ -0,0 +1,16 @@
+package com.carina.methods.SauceDemo.commons;
+
+import com.carina.methods.SauceDemo.components.ProductInCart;
+import com.carina.methods.SauceDemo.screens.CheckoutScreen;
+import org.openqa.selenium.WebDriver;
+
+public abstract class CartBaseScreen extends ScreenBaseClass {
```

Comment:
please use following formatting rules: `Base` at the end
```suggestion
public abstract class CartScreenBase extends ScreenBaseClass {
```